src/tools/miri/src/bin/miri.rs

```diff
@@ -436,7 +435,7 @@ fn parse_range(val: &str) -> Result<Range<u32>, &'static str> {
     Ok(from..to)
 }
 
-fn main() {
+fn main() -> ExitCode {
```

Comment:
We have a custom `exit` function in this file that does some custom stuff. It's probably better to keep calling that here instead of changing the logic.